community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -971,22 +1110,27 @@ public PrimitiveIntIterator graphGetPropertyKeys( KernelStatement state )
     }
 
     @Override
-    public Property graphGetProperty( KernelStatement state, int propertyKeyId )
+    public boolean graphHasProperty( KernelStatement state, int propertyKeyId )
+    {
+        return graphGetProperty( state, propertyKeyId ) != null;
+    }
```

Comment:
Those methods are there for consistency mostly. We should revisit this in 3.0 and decide what we want to do with it.